### DISCUSSION TITLE
friend remove

### DISCUSSION THREAD

**Discussion by windosws** [Rapptz/discord.py#7679] [Q&A] [answered]
https://github.com/Rapptz/discord.py/issues/4004#issue-600370979

so right here someone answered the issue with a code who can remove friend, but its removing ALL friends and a specific one with id or @ 
someone can fix that ?

**Comment by JDJGInc** [comment]
remove_friend is a self bot thing, and is not supported.

**Comment by imvaskel** [comment, marked as answer]
discord.py does not support user bots anymore.

**Reply by windosws** [reply]
yea thats why i still have a selfbot who can exploit discord but well u didn't answer to my question.

**Reply by SebbyLaw** [reply]
> yea thats why i still have a selfbot who can exploit discord but well u didn't answer to my question.

Self-bots are not supported by discord.py. Your question is on the discord.py repository discussions board. In the context of these two facts, you were answered sufficiently.

**Reply by ChesterWOV-DPY** [reply]
By the way self-bots and user-bots violates the Discord TOS.
